biogrid/apps/biogrid-mvp/src/assets/icons/house.svg

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"

```

Comment:
I would recommend running all svgs through a public svg compressor

Reply:
Did you try running these through a compressor? That would probably get rid of the empty <g> tags, etc.